src/Node/GethNode.ts

```diff
@@ -44,18 +44,25 @@ export default class GethNode extends Node {
    * Read the bootnodes from the utility chain subdirectory.
    */
   public readBootnodes(): void {
-    this.logInfo('reading bootnodes from disk');
-    this.bootnodes = fs.readFileSync(
-      path.join(
-        Directory.projectRoot,
-        'utility_chains',
-        `utility_chain_${this.chain}`,
-        'bootnodes',
-      ),
-      {
-        encoding: 'utf8',
-      },
-    );
+    // Added try catch, because this is called even in case of mosaic stop.
+    // This is needed only while mosaic start.
+    try {
+      this.logInfo('reading bootnodes from disk');
+      this.bootnodes = fs.readFileSync(
+        path.join(
+          Directory.projectRoot,
+          'chains',
+          this.originChain,
+          this.chain,
+          'bootnodes',
+        ),
+        {
+          encoding: 'utf8',
+        },
+      );
+    } catch (e) {
+      this.logInfo('Boot nodes not present');
```

Comment:
I think just catching and logging is not enough. If boot nodes are not found, it can't start node.

Reply:
This is called for almost from all the commands, including stop.
There will be an architecture change.

Reply:
Should there be a ticket to fix this?